platform/ui/src/components/SegmentationGroupTable/SegmentationConfig.tsx

```diff
@@ -13,6 +13,15 @@ const ActiveSegmentationConfig = ({
   setFillAlpha,
   usePercentage,
 }) => {
+  const [
+    useOutlineOpacityPercentage,
+    setUseOutlineOpacityPercentage,
+  ] = useState(usePercentage);
+
+  const [useFillAlphaPercentage, setUseFillAlphaPercentage] = useState(
+    usePercentage
+  );
```

Comment:
This makes sense, but why we don't do this for the `setOutlineWidthActive` and `setOutlineOpacityActive` too?

Reply:
I am not entirely sure what you mean here. 

I double checked the diffs of this file and I have done something similar for each of...
- `setOutlineOpacityActive`
- `setFillAlpha`
- `setFillAlphaInactive`

Each of the above key off of `usePercentage` and use `InputRange` so that makes sense.

However `setOutlineWidthActive` does not appear to key off of `usePercentage` nor does it use `InputRange`. Am I missing something?

Reply:
I see

Reply:
Are we good then?

Reply:
yes